src/site/topic16.rst

```diff
@@ -2,6 +2,20 @@
 Topic #16 --- Exceptions
 ************************
 
+* We've seen a number of exceptions at this point
+* We should also be familiar with them at a fairly high level
+* We should know how to use them and handle them
+
+.. image:: img/exceptions.png
+   :width: 500 px
+   :align: center
+   :target: https://www.tutorialspoint.com/java/java_exceptions.htm
```

Comment:
Copyright ok on this? Seems ripped straight from the site.

Reply:
Defo straight from them. Could remake, but perhaps I'll wait for a cease and desist. 

Either way #11.